packages/remix-react/links.ts

```diff
@@ -91,16 +86,6 @@ export interface HtmlLinkDescriptor {
    */
   sizes?: string;
 
-  /**
-   * Images to use in different situations, e.g., high-resolution displays, small monitors, etc. (for rel="preload")
-   */
-  imagesrcset?: string;
-
-  /**
-   * Image sizes for different page layouts (for rel="preload")
-   */
-  imagesizes?: string;
-
```

Comment:
Removing this will technically be a breaking change I think 🤔😕

Reply:
Yeah, I can add these back and just mark them as `@deprecated`

Reply:
We can remove them in Remix v2 👍

Reply:
Appended here so they are easy to remove later https://github.com/remix-run/remix/blob/6e5a89c97a5a67bcd135cce88c9d0787ec7b3e08/packages/remix-react/links.ts#L186